app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/templates.tsx

```diff
@@ -1,62 +1,151 @@
IA5XSydgSM/liuZgVGIa8MQYPGzx3iSvIfdyMnbODjt8qtu7dBb+9QyiSJgCAeYFcDR3AmilIXy7n1G+fvRQuRDarFpC27rock8iNz/AL+VASyHKwgBYpVooSdM8/z6UUrgIMbf1oa5tZFLtblbmFTgMBuPpzrlnOz+SNgJB/3cnL5A073nGSJG2k8RnHIScAj86Mi4pDbRFPDMpIxgkYpb4C3Ca0k0sM6lPT0q6HhnjIrRy7no3X5Vo11LDDGTto7AcgSZ4pMX1IQnbT0qa8Xu1x+MTvncDeqm4ZKqao1eUDZigBCn17UHIDGxDYz86qDIw4kpRlhtxf3F0U8aTUE5DAFdjunU5zkmgA9dEhrTM2mOkul06cKcncGuOAEHiFlHMAdKVx6mbbejriaWVUDoVVFAwBgetLOAY0HjmTimKAKiLnPxHmaNRiyqGOGxuV60rVsY3oqOXHNtq7GYBfEYRO0TqckjrRksqvFr04xz070tjuoCuiN2jfudxREUcqjxPiQ/voSVNdtHmAbGl5lkZsKAox0o+xu5LPGJPL2PKlr3CKFZQCRk5HWl11fF/hNH0wwgdUqeDNrcy23ELMyW90I5sYKs2Bq9T0rPX1wxIhm1BsZeMrg57k9aQC4k16tRzRIu8xaZZMhfMuo8u9YtOyMfUF/HMLSR/EDFsYOR6UZ79/5cR+YNZ27vAz4ibSp6E70L4x/tPua4qD3gKWHaJ7eKFLh1nYYOfKdxnpV9wIBOEAwHVgpzj5d66cEABW16id15Z6Gh7rW0iFceUnH2rzVAM+iKweW3jGJB8ZyPLyFDku1wAN1UdeXzo+bSfLkai2cDvVVoFlkAQHljzbDIO9PBXGDFHMacLMdu2qRTqIxy2AFGpIiNMNTFc6sgDr1xSt42jljDDT8W+djyotCgWXUSQw2A7UtlE1QcwpblTqZoxsNJU/vAc6FvgHjMkLMVOQCe1RIBZJWYAKxBLbAj+vWiLa0mWLXExe3ckDPMZ5ff+dcuBNcHHMXwRtBbrLlVJG+e1DyXNxFFPblwYpyC6sBuRvt2/SjZIBJbtBuSBzHPHWqngWZGA+JVyAN8Gn8HvEAkScU+hi6eSTyppGw6Z/maKuQj25OVSTOzAZDDBG/brS/h7kSoWTXpwQMZJ612bxijSKrBjI5C4IK77VLdUrHIj1s4l0d8IJhr82tcsEbIYn0/3yoq2uxE48U6opW5EgY2z+tIkDTTbI+picYHXtRbWzTg+JrjI+DUhA9elT3CleG8zlZ/E1aSt4rFHZNY0ko2kspyMEg7jnse9LrnhsccJkgXxFA3RSQw+Q32pZDJeWyhYmDL1VmzvnNGi/laNVCPG648yb1Imoepvi2RNsRXHI5nbX9l+OFnNwVzgjIB/lU76KCJPFsVSRD3ySv9KgeKJu17Ck3cmNdQx8zRlt+zL8ArazW7t1QkfyJFWDXqDuYmTHTsRhRFUbuVKs5x2zVsYkznLemKaT8E8NlkhnkxnbUhJ+2aHubKRW/7VMdWEZG/XanfqGmY8NEnT3ftMHjvNGdTFiOYIFWi5SU7x7em1Vjh4DBhLG4yM4Jz9xR9raxIPNKm+2G5k9PsKpF1XdWiHrccESCqgGdwvYkH71736S3OYWKnuDXLqezKZDeGy7YzsT8qQz3chkAEbaeZOKcL0xyYhdM5bgRxNf5J655nODQNzfRwqPMpZuQzQxzPEFKSEE76cCqRZW+rTKZA3qc9aUdYg4BlS6Ty09JxeUYKiMD6k1KPihfUJQw7FcVVNaQLII1Z9RGR+Gf+ftUTYkcy6/8A1N+lIfVAnG7EoWhMfTD1ubU480gPdgK77xB/bH/L/rS9rCcEadx/iUqfvXvcrn+Bfzqf3OP8kZ0FP9ksa6yhBkAOcFl+eaquAypqabWCdvl0oZGJQ/PHLpVsqE24DMSOeO1Ar84lpHEotZkW4UsQq6wck8t6NuHFs7+YCJ90x0Poe+PtQV3ZiOGJ45CNSZIxVDnVHpJOMA7nrTlOYJWPuH3ZmtSnPzbam5duXWj7KWOT8NsMWGzAf77VneFW+opmRwmd1XbO9aaxt7WEKyRP5gcgyfLsKTZqBWeYXSJHElcWi3BKqdRbljmPlRHA5RbzmxuB+HINGOx2xn6VYbiOEt4UOnT5QdWTv86X3xiXdode55uwH5A4pY9RrxjEW+nY45hF2onuESJCLhT5nJA5cjz54zy54odeBtLOXHEIomzkKgY/PH6VRbyQLDPILWPCLnTzDEnG/wCf2FBw3Utu4EJ0odyoAwf60q3UW2D4HE1ERPqGZrLbhMVto8WfxEU51Sat/lhht9KOEvDEO6xBidvJsMfPnWL/AGnKQXOc45asg/Q5odOMXLHy6Qi76CoOfrjNed7Sy35M8f7lV4VZqr2SzjmDWvDOGkruHZWHLrjA/nVUNzLO2vwLbBGSUDD+fOk0fGZJocPBEW6M2+PlU4md8MG0gn4QB1rbNOEGCeYIuJPaOQkcjlpLWQkDmEZf6UvuwiSl4oLxs7DRuo+hSoT8OUYZpGYYzjJH9aAYgoMg6MkFQxH8jWVoAczGcHiXarsuBJaXGgHJZYWAP1xVkUlwinVFcBf4ZJcD8udWxoyv+FIUJXUCOlGB5tP4kpbIyMDHSjss47QM/mKnuZ8HRCw3xsTj+Rr0a3GstplydiUV804R9WzZz3zXD5ZCgJ5c80oX47LNIJ7mUR2czAE3JXbYEEH61x7UnZ5DIB0A5VJnZiV1sPMOtSJO2rBHbegNlmc5hACVPIAoR4i2nbzpVLPGTn3VQcY2TOaYsg8JipKnYDB5VW1sHYgORjntnNDv+82LXMTAZtkxnrHXQYVyPDZRj91f0onwfKckH6VY8a6T5Rt6UzqYncwES2vLL7DcnpVbz2Z/8SNuhA/pTMwJgjGw6fOg5oIQw/D59iRRLYp75nEmCe+Km+qI9iQ1e/aH/t/kf1q5bON84Zxz21Zq0cIQgfin/LTv6Zg5In//2Q==`
```

Comment:
Never ever hardcode binary files in the repo. This is a no-go for merging this.

Reply:
`borderBgImg` is not a binary data, it is a tiny SVG text. In such case its ok

Reply:
note: you can just set the text color in css, and then use `currentColor` in svg to make the svg the same color as the text